lib/opensusebasetest.pm

```diff
@@ -29,6 +30,53 @@ sub export_logs {
     select_console 'root-console';
     save_screenshot;
 
+    # check if wicked is installed and turn on debug, get all needed wicked logs if network interface is down
+    if (script_run 'rpm -q wicked') {
```

Comment:
As there is no else I guess it is way easier to read if you check on the script result and call a properly named sub accordingly, i.e. something like
```perl
sub export_wicked_logs {
…
}
...
my $wicked_installed = script_run 'rpm -q wicked';
export_wicked_logs if $wicked_installed;
…
```
if you like it